.github/ISSUE_TEMPLATE/config.yml

```
@@ -1,5 +1,8 @@
 blank_issues_enabled: false
 contact_links:
   - name: Question about wgpu
-    url: https://github.com/gfx-rs/wgpu/discussions/new
+    url: https://github.com/gfx-rs/wgpu/discussions/new/choose
     about: Any questions about how to use wgpu should go here.
+  - name: Security concerns
+    url: https://github.com/gfx-rs/wgpu/security
+    about: Information about reporting possible vulnerabilities.
```

It would be good to give a bit more what classifies as a security issue here as this is a decision point:

```suggestion
    about: Information about reporting possible vulnerabilities. For example: use after free, buffer overruns, etc; panics and null pointers dereferences however are not security sensitive.
```

---

Links to some encyclopedic content to build intuition would also be neat! 😀

---

Since these links are displayed in-line nearly identically to issue templates, I think it would be good if the description was more aligned with _what the user is possibly trying to do_. Something like:

> If you have found a possible vulnerability in wgpu, please read this security policy to understand how to report it.

---

SGTM